README.md

```diff
@@ -1,4 +1,5 @@
-[![Build status](https://ci.appveyor.com/api/projects/status/b6qjwi71527ifh0g/branch/master?svg=true)](https://ci.appveyor.com/project/HEskandari/farsilibrary/branch/master)
+[![Build status](https://ci.appveyor.com/api/projects/status/b6qjwi71527ifh0g/branch/master?svg=true)](https://ci.appveyor.com/project/HEskandari/farsilibrary/branch/master) | [![GitHub version](https://badge.fury.io/gh/ghost1372%2FFarsiLibrary.svg)](https://badge.fury.io/gh/ghost1372%2FFarsiLibrary)
```

Comment:
Why is this needed? It is pointing to the wrong branch.

Reply:
ok